packages/edit-site/src/utils/get-item-title.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { decodeEntities } from '@wordpress/html-entities';
+
+// TODO: This function is duplicated in packages/editor/src/dataviews/actions/utils.ts,
+// so they should be consolidated.
+
+export function getItemTitle( item ) {
+	if ( typeof item.title === 'string' ) {
+		return decodeEntities( item.title );
+	}
+	if ( item.title && 'rendered' in item.title ) {
+		return decodeEntities( item.title.rendered );
+	}
+	if ( item.title && 'raw' in item.title ) {
+		return decodeEntities( item.title.raw );
+	}
+	return '';
+}
```

Comment:
I needed to duplicate the `getItemTitle` function to avoid importing it from the fields package into edit-site.